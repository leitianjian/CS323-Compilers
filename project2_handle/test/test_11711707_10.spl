struct test {
    int a;
    char c;
} test (int d, char b) {
    struct test t;
    t.a = d;
    t.c = b;
    return t;
}