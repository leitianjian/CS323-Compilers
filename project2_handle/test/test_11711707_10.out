Error type 16 at Line 1: define a structure in return type
Error type 17 at Line 6: undefined structure type
Error type 5 at Line 6: unmatching type on both sides of assignment
Error type 17 at Line 7: undefined structure type
Error type 5 at Line 7: unmatching type on both sides of assignment
